metadata.json

```diff
@@ -22,13 +22,13 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "14.04"
+        "18.04"
```

Comment:
do you know if Ubuntu 16.04 / Debian 9 work as well? If so, please also add them.

Reply:
Hi Tim, you're right !
PR just updated :-)